proteus/PostProcessingTools.py

```diff
@@ -622,6 +675,7 @@ def __init__(self,vectorTransport=None,vtComponents=[0],omitFluxBoundaryNodes=Tr
                 self.solutionTestSpaceIsNotPWL[ci] = True
                 logEvent("pwl post-processing for finite element space"+str(self.vt.u[ci].femSpace))
                 #need to set up w and w*dS_f and weights to build residuals for linears out of R_{e,i}
+                #Point1
                 if self.testSpace == None:
```

Comment:
While we're updating this file it would be good to replace the `== None` tests with `is None` so we don't generate so many warnings.